CONTRIBUTING.md

```diff
@@ -1,21 +1,83 @@
 # Contributing
 
-Our Helm Chart's README.md is automatically
-generated using the pre-commit hooks for
+<!-- toc -->
+
+- [General](#general)
+- [Quickstart to Skaffold in Minikube](#quickstart-to-skaffold-in-minikube)
+- [pre-commit Hooks](#pre-commit-hooks)
+- [Localized Deployments (for internal-testing)](#localized-deployments-for-internal-testing)
+  - [minikube](#minikube)
+  - [skaffold](#skaffold)
+    - [profiles](#profiles)
+    - [Container Images Stored in Private Repositories](#container-images-stored-in-private-repositories)
+      - [Google Artifact Repository](#google-artifact-repository)
+      - [Docker Hub](#docker-hub)
+  - [Accessing the k8s Resources](#accessing-the-k8s-resources)
+    - [Ingress](#ingress)
+    - [Ingress - Login](#ingress---login)
+    - [Port Forward to the `teams-app` Service](#port-forward-to-the-teams-app-service)
+    - [Port Forward to the `teams-api` Service](#port-forward-to-the-teams-api-service)
+    - [Port Forward - Login](#port-forward---login)
 
-* [https://github.com/norwoodj/helm-docs](https://github.com/norwoodj/helm-docs)
-* [https://github.com/Lucas-C/pre-commit-hooks-nodejs](https://github.com/Lucas-C/pre-commit-hooks-nodejs)
+<!-- tocstop -->
 
-1. Install dependent tools
-    * Using [asdf](https://asdf-vm.com/)
+## General
+
+1. Install tool dependencies
+    1. Install
+       [asdf](https://asdf-vm.com/)
+    1. Install tools managed by `asdf`
 
         ```shell
         make asdf
         ```
 
-    * Manually
-      * [helm-docs](https://github.com/norwoodj/helm-docs#installation)
-      * [pre-commit](https://pre-commit.com/#installation)
+## Quickstart to Skaffold in Minikube
+
+1. Auth with gcloud
+
+    ```shell
+    gcloud auth application-default login
+    ```
+
+1. Install the asdf tools
+
+    ```shell
+    make asdf
+    ```
+
+1. In one terminal, start minikube
+
+    ```shell
+    make start
+    ```
+
+1. Set skaffold secrets.
+   See [skaffold](#skaffold)
+
+1. Run skaffold
+
+    ```shell
+    make dev-keep
+    ```
+
+1. In another terminal, run minikube tunnel (and provide your password when prompted)
+
+    ```shell
+    minikube tunnel
+    ```
+
+1. Navigate to
+   [https://local.fiftyone.ai](https://local.fiftyone.ai)
+   and login
```

Comment:
This is super nice and very clear to get moving.  I like it a lot, thankyouverymuch!